[Contact] 코드 리팩토링 및 버그 수정

ingbinsee opened the pull request:
## 🔍️ 이 PR을 통해 해결하려는 문제가 무엇인가요?

> 어떤 기능을 구현한건지, 이슈 대응이라면 어떤 이슈인지 PR이 열리게 된 계기와 목적을 Reviewer 들이 쉽게 이해할 수 있도록 적어 주세요
> 일감 백로그 링크나 다이어그램, 피그마를 첨부해도 좋아요

- margin 중첩현상 해결
- TitleSection 컴포넌트 depth 정리
- 폴더명 변경(contact > Contact)으로 인한 경로 에러 해결

## ✨ 이 PR에서 핵심적으로 변경된 사항은 무엇일까요?

> 문제를 해결하면서 주요하게 변경된 사항들을 적어 주세요

- margin 중첩현상을 padding으로 해결
- 분리되어 있던 ContactTitleInner 컴포넌트를 ContactTitleSection 컴포넌트에 병합
- 폴더명 변경에 따른 import 경로명 변경

## 🔖 핵심 변경 사항 외에 추가적으로 변경된 부분이 있나요?

> 없으면 "없음" 이라고 기재해 주세요

- 없음

## 🙏 Reviewer 분들이 이런 부분을 신경써서 봐 주시면 좋겠어요

> 개발 과정에서 다른 분들의 의견은 어떠한지 궁금했거나 크로스 체크가 필요하다고 느껴진 코드가 있다면 남겨주세요

- 없음

## 🩺 이 PR에서 테스트 혹은 검증이 필요한 부분이 있을까요?

> 테스트가 필요한 항목이나 테스트 코드가 추가되었다면 함께 적어주세요

- 없음

### 📌 PR 진행 시 이러한 점들을 참고해 주세요

- Reviewer 분들은 코드 리뷰 시 좋은 코드의 방향을 제시하되, 코드 수정을 강제하지 말아 주세요.
- Reviewer 분들은 좋은 코드를 발견한 경우, 칭찬과 격려를 아끼지 말아 주세요.
- Review는 특수한 케이스가 아니면 Reviewer로 지정된 시점 기준으로 3일 이내에 진행해 주세요.
- Comment 작성 시 Prefix로 P1, P2, P3 를 적어 주시면 Assignee가 보다 명확하게 Comment에 대해 대응할 수 있어요
  - P1 : 꼭 반영해 주세요 (Request Changes) - 이슈가 발생하거나 취약점이 발견되는 케이스 등
  - P2 : 반영을 적극적으로 고려해 주시면 좋을 것 같아요 (Comment)
  - P3 : 이런 방법도 있을 것 같아요~ 등의 사소한 의견입니다 (Chore)

#

---

## 📝 Assignee를 위한 CheckList

- [ ] To-Do Item

ingbinsee committed:
🐛 [Fix] : 버그 수정

+ margin 중첩현상을 해결하였습니다.
+ 분리되어 있던 ContactTitleInner 컴포넌트를 ContactTitleSection 컴포넌트에 병합하였습니다.
+ 폴더명 변경에 따른 import 경로명을 변경하였습니다.

ingbinsee committed:
♻️ [Refactor] : 스타일링 코드 수정 및 키보드 접근성 개선

- peer를 활용하여 지정한 스타일을 checked 속성을 활용하는 방법으로 변경하였습니다(클릭 시 글자색, 배경색이 변경).
- 문의유형 input을 hidden으로 설정하여 키보드 접근성이 지켜지지 않았으나 CSS 변경과 focus-within을 활용하여 키보드 접근성을 개선하였습니다.

kelvin-chihyun reviewed (approved):
확인했고, 이상없습니다! 고생하셨습니다.

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

dani-day reviewed (approved):
이견 없습니다 👍

ingbinsee committed:
♻️ [Refactor] : 리뷰 사항 반영

+ 문의유형에 props로 onClick 이벤트와 checked를 넣어두었습니다.
+ 정규표현식을 활용하여 연락처 input에 문자 입력 시 에러 메시지가 보이도록 하였습니다.
+ 정규표현식을 활용하여 연락처 input에 자동으로 하이픈(-)이 입력되도록 하였습니다.

! 모바일에서의 문의유형 아이콘은 추후 디자인 변경 방향에 따라 개선될 예정입니다.
